src/Sylius/Bundle/PromotionBundle/DependencyInjection/Configuration.php

```diff
@@ -90,6 +93,23 @@ private function addResourcesSection(ArrayNodeDefinition $node): void
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('catalog_promotion')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->variableNode('options')->end()
+                                ->arrayNode('classes')
+                                    ->addDefaultsIfNotSet()
+                                    ->children()
+                                        ->scalarNode('model')->defaultValue(CatalogPromotion::class)->cannotBeEmpty()->end()
+                                        ->scalarNode('interface')->defaultValue(ResourceInterface::class)->cannotBeEmpty()->end()
```

Comment:
Wasn't `interface` deprecated?

Reply:
Indeed https://github.com/Sylius/SyliusResourceBundle/pull/97 🖖 However, to keep consistency within the bundle, I think we can leave it and remove it altogether with other's when we upgrade to ResourceBundle 2.0

Reply:
But still, it should be `CatalogPromotionInterace::class` 💃